apps/website-25/src/components/Nav.tsx

```diff
@@ -66,36 +67,47 @@ const ExploreSection: React.FC<{
 
 const NavLinks: React.FC<{
   exploreSectionInline?: boolean;
-  children: React.ReactNode;
   courses: Array<{ title: string; href: string; isNew?: boolean }>;
   exploreExpanded: boolean;
   onToggleExplore: () => void;
   className?: string;
+  isScrolled: boolean;
 }> = ({
-  exploreSectionInline, children, courses, exploreExpanded, onToggleExplore, className,
-}) => (
-  <div className={clsx('nav-links flex gap-9', className)}>
-    <div>
-      <button
-        type="button"
-        onClick={onToggleExplore}
-        className="nav-links__dropdown-button flex items-center gap-2 hover:text-bluedot-normal"
-      >
-        Explore
-        <DropdownIcon expanded={exploreExpanded} />
-      </button>
-      {exploreSectionInline && (
-        <ExploreSection
-          expanded={exploreExpanded}
-          courses={courses}
-          className="nav-links__explore-section"
-          innerClassName="pl-6 pt-6"
-        />
-      )}
+  exploreSectionInline,
+  courses,
+  exploreExpanded,
+  onToggleExplore,
+  className,
+  isScrolled,
+}) => {
+  const navLinkClasses = clsx('nav-link-animation', isScrolled && 'nav-link-animation-dark');
+
+  return (
+    <div className={clsx('nav-links flex gap-9 [&>*]:w-fit', className)}>
+      <div>
+        <button
+          type="button"
+          onClick={onToggleExplore}
+          className={clsx('nav-links__dropdown-button flex items-center gap-2', navLinkClasses)}
+        >
+          Explore
+          <DropdownIcon expanded={exploreExpanded} />
+        </button>
+        {exploreSectionInline && (
+          <ExploreSection
+            expanded={exploreExpanded}
+            courses={courses}
+            className={clsx('nav-links__explore-section', isScrolled && '[&_a]:link-on-dark')}
```

Comment:
i'd recommend passing isScrolled so that all the ExploreSection logic is contained in a single place:
```suggestion
            className='nav-links__explore-section'
            isScrolled={isScrolled}
```
```
// ExploreSection
...
   <a key={course.href} href={course.href} className={clsx('nav-explore-section__dropdown-link', isScrolled && 'link-on-dark')}>
```
for example, if there is a change we need to make to the link logic on scrolled, i would expect to find these styles applied directly to the element. and then we don't need to repeat it in the mobile & desktop implementations